### PR TITLE
Modified installation from sources section. [5995]

### PR DESCRIPTION
**Pull request by richiware**

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/23/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/24/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/25/)

File: docs/sources.rst [hunk 1]
```diff
@@ -1,23 +1,94 @@
...
+************
 
-    $ git clone https://github.com/eProsima/Fast-RTPS
-    $ mkdir Fast-RTPS/build && cd Fast-RTPS/build
+Asio and TinyXML2 libraries
+===========================
 
-If you are on Linux, execute: ::
+On Linux, you can install these libraries using the package manager of your Linux distribution.
+For example, on Ubuntu you can install them using its package manager with the next command.
```
Comment (raquelalvarezbanos) [inline docs/sources.rst]:
```suggestion
For example, on Ubuntu you can install them by using its package manager with the next command.
```

File: docs/sources.rst [hunk 2]
```diff
@@ -1,23 +1,94 @@
...
+* asio.1.12.1.nupkg
+* tinyxml2.6.0.0.nupkg
+
+Once these packages are downloaded, open an administrative shell and execute the following command:
+
+.. code-block:: batch
+
+   choco install -y -s <PATH\TO\DOWNLOADS\> asio tinyxml2
+
+Please replace ``<PATH\TO\DOWNLOADS>`` with the folder you downloaded the packages to.
```
Comment (raquelalvarezbanos) [inline docs/sources.rst]:
```suggestion
Please replace ``<PATH\TO\DOWNLOADS>`` with the folder where you downloaded the packages.
```

File: docs/sources.rst [hunk 3]
```diff
@@ -1,23 +1,94 @@
...
+
+Colcon installation
+*******************
+
+colcon_ is a command line tool to build sets of software packages.
+We show you how to use it to compile easily Fast-RTPS and its dependencies.
```
Comment (raquelalvarezbanos) [inline docs/sources.rst]:
```suggestion
This section explains to use it to compile easily Fast-RTPS and its dependencies.
```

File: docs/sources.rst (hunk 2 above)
Comment (richiware) [reply docs/sources.rst]: As discussed this is fine.

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/26/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/27/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/28/)

**Pull request merged**